R/02-template-Renviron.R

```diff
@@ -0,0 +1,13 @@
+# This template contains the format of the environment variables that are mentioned in the template:
```

Comment:
The .Renviron should be at the root of the R session and would be ignored otherwise. Maybe we should then place the template at root also?

I think a better option would be to just document this in the template-db.R and in the documentation. Hence don't have this as template file.

Another Option would be to generate these files, instead of providing templates. This would be in the spirit of [usethis](https://cran.r-project.org/web/packages/usethis/index.html) and maybe much easier because we can add entries to the ..Renviron, make sure it is in the gitignore, and properly setup the database template.

Reply:
As just discussed, I will go for the second option and prepare a function that adjusts all which is required to add a database.
@wahani Thanks for this valuable feedback.